app/queries/update_school_links_mutator.rb

```diff
@@ -0,0 +1,38 @@
+class UpdateSchoolLinksMutator < ApplicationQuery
+  include AuthorizeSchoolAdmin
+  property :id, validates: { presence: true }
+  property :title,
+           validates: {
+             length: {
+               minimum: 1,
+               maximum: 24,
+               message: 'InvalidLengthTitle'
```

Comment:
I think you can ignore the message part as the users can't see it / or we should have a more readable error message.